.circleci/config.yml

```diff
@@ -314,6 +319,72 @@ jobs:
             ./cc-test-reporter upload-coverage --input coverage/codeclimate_full_report.json
 
             ruby scripts/check_coverage.rb ${PWD}/coverage/codeclimate_full_report.json
+  update-staging:
+    docker:
+      - image: cimg/aws:2023.01
+    steps:
+      - checkout
+      - setup_remote_docker
+      - aws-cli/setup:
+          aws-region: STAGING_AWS_REGION
+          aws-access-key-id: CIRCLE_CI_USER_ACCESS_KEY_ID 
+          aws-secret-access-key: CIRCLE_CI_USER_SECRET_ACCESS_KEY
+      - run: 
+          name: build searchgov image
+          command: |
+            docker build --build-arg AWS_ACCESS_KEY_ID=${CIRCLE_CI_USER_ACCESS_KEY_ID} \
+            --build-arg AWS_REGION=us-east-1 \
+            --build-arg AWS_BUCKET=dgsearch-images \
```

Comment:
@sb-ebukaanene This doesn't seem appropriate for a staging environment.  We currently use `dgsearch-images-dev` in staging stacks.  We don't want staging stacks to have access to production assets, however that is accomplished.  If we're changing this up to have parallel naming in staging and prod just controlled by regions we'll likely need to thoroughly test it, since I suspect the code assumes a `dev` bucket in non-production envs (but I could be wrong).  How is this handled, now?  This is tied in to https://cm-jira.usa.gov/browse/SRCH-5229

Reply:
@lorawoodford  i changed the bucket to the dev bucket for staging